modules/ROOT/pages/authentication-authorization/database-administration.adoc

```diff
@@ -853,6 +853,10 @@ For example, to grant the role `regularUsers` the ability to create new properti
 GRANT CREATE NEW PROPERTY NAME ON DATABASE neo4j TO regularUsers
 ----
 
+[NOTE]
+====
+The `SHOW PRIVILEGES` commands return the `NAME MANAGEMENT` privilege as the action `token`, when not using `AS COMMANDS`.
```

Comment:
with the new structure of the section it might look better to put the note directly after `Table 28. Node label, relationship type, and property name privileges management syntax` and before the example?